src/android/indooratlas.gradle

```diff
@@ -4,8 +4,11 @@ repositories {
         url "http://indooratlas-ltd.bintray.com/mvn-public"
     }
 }
+}
 
 
 dependencies {
+    compile 'com.android.support:appcompat-v7:27.0.2'
     compile 'com.indooratlas.android:indooratlas-android-sdk:2.6.0@aar'
+    compile 'com.indooratlas.android:indooratlas-android-wayfinding-sdk:0.1.1-dev-5@aar'

```

Comment:
This should be `indooratlas-android-wayfinding` (without the `-sdk`). The SDK version should be changed to 2.7

Reply:
It has now been published: https://bintray.com/indooratlas-ltd/mvn-public-beta/indooratlas-android-wayfinding/2.7.0-beta-9